src/routes/auth.route.js

```diff
@@ -11,85 +11,16 @@ import {
 
 const authRouter = Router();
```

Comment:
So all the route documentation were actually destined to eventually `die`? because you have removed everything now. I thought they would always be here

Reply:
Oh, yeah, I had to **move** it not **remove**. The documentation in those routes files were mostly duplicates and not properly documented. 
So to avoid duplication, I created a new file called: `documentation.route.js` that stores all the documentation for the routes.